examples/with-nextjs-and-clerk-auth/src/themes/monite.ts

```diff
@@ -7,6 +7,8 @@ import type {
 import type { Theme } from '@mui/material/styles/createTheme';
 import type { TypographyOptions } from '@mui/material/styles/createTypography';
 import { deepmerge } from '@mui/utils';
+import '@mui/x-data-grid/themeAugmentation';
```

Comment:
optional: could be written more explicitly
`import type {} from '@mui/x-data-grid/themeAugmentation';`

Reply:
When import is written this way, the Intellij will optimize it away